src/libraries/restore/netcoreapp/netcoreapp.depproj

```diff
@@ -11,6 +11,17 @@
     <PackageReference Update="Microsoft.NETCore.App" IsImplicitlyDefined="false" PrivateAssets="None" />
   </ItemGroup>
 
+  <!-- Mark framework references from previous netcoreapp tfms >= 3.0 as copy-local. -->
+  <Target Name="_FrameworkReferencesToPrivate"
```

Comment:
Instead I’d just do this for all items in depproj targets.

Reply:
Can you elaborate? Not sure I follow. Do you mean to actually Define an ItemDefintionGroup for reference?

Reply:
If you mean you would do it for all targets, I would rather not as we include netstandard2.0 here as well which just creates noise in the ref folder, ie dotnet-XmlGenerator.

Reply:
I meant that the purpose of DepProj is to copy.  It flows these items to content here: https://github.com/dotnet/runtime/blob/8f679991b8fc9494dc548620f48f841d38a6dc24/eng/depProj.targets#L95

This is effectively doing the same thing you're trying to do (force to copy).  If that wasn't working for framework references we should fix it there.

Reply:
Actually I investigated why this wasn't working with the depproj/binplacing logic. I will submit a follow-up PR when I have some cycles to fix it there.